src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,47 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd(newMovie: Movie): void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
```

Comment:
The other way to type props is that you can type the component object like this. It's not a mistake just a new info for you
```suggestion
export const NewMovie = ({ onAdd }: Props) => {
```